_posts/2019-09-15-guide_create_surf_ramps.md

```diff
@@ -123,11 +123,20 @@ For curved ramps this means that the portion near the bottom of the ramp where s
 The standard fix for this has been to add a flat "base" portion at the bottom of the ramp segment that you can't surf on. 
 This means that any precision errors would only take place at the base of the ramp and not the surfable portion.  
   
+This is explained in more depth <a href="https://youtu.be/jcC2UKydb9Q" target="_blank">here</a>.  
```

Comment:
Any reason to not use the markdown style for links? HTML is great but markdown is much nicer to read. https://guides.github.com/features/mastering-markdown/

HTML should really only be used when necessary (custom styling). Markdown is pretty powerful for a lot of use cases.

Reply:
The original idea was to have the external links open in a new tab, but there is a way to do that through markdown so i will remove the html

Reply:
fixed in commit [dab0b8c](https://github.com/momentum-mod/docs/pull/20/commits/dab0b8ccdd73b788cb638b3e8e4ec5e020035fc0)